cli/cmd/root.go

```diff
@@ -43,13 +42,13 @@ func NewRootCmd() *cobra.Command {
 	// Set output of cmd.Print to stdout. (By default, it's stderr.)
 	rootCmd.SetOut(os.Stdout)
 
-	rootCmd.PersistentFlags().String("config", constants.ConfigFilename, "path to the configuration file")
-	must(rootCmd.MarkPersistentFlagFilename("config", "yaml"))
-
+	rootCmd.PersistentFlags().String("workspace", "", "path to the Constellation workspace")
```

Comment:
```suggestion
	rootCmd.PersistentFlags().StringP("workspace", "C", "", "path to the Constellation workspace")
```

The short flag `-C` is a convention used by many tools (including make, tar, [`go`](https://github.com/golang/go/issues/50332) and many others).